static/sb-styles/preview.css

```diff
@@ -2,7 +2,7 @@
  * Overrides for the Storybook preview iframe.
  */
 html {
-    font-size: 50%;
+    font-size: 62.5%;
```

Comment:
This change isn't registering in Chromatic yet! It still shows 50% and a bunch of stuff looks off. Weird! https://5e1bf4b385e3fb0020b7073c-jxifgupglv.chromatic.com/sb-styles/preview.css

Reply:
I'm seeing it as 62.5% now! https://5e1bf4b385e3fb0020b7073c-ohuxdtgckx.chromatic.com/?path=/story/packages-banner--default 

![image](https://github.com/user-attachments/assets/1d6543e2-ff35-4fe1-820e-316dfcd8716f)

The typography in the docs are quite large though, is this expected? My zoom is at 100%! https://5e1bf4b385e3fb0020b7073c-ohuxdtgckx.chromatic.com/?path=/docs/overview--docs

![image](https://github.com/user-attachments/assets/403ba312-f157-4a35-94b3-7aea7f802b73)

Reply:
Ah yes, good catch! I fiddled with those numbers some more and it's looking better to me. There is a slight shift on the overview page, possibly from a dynamic width or height somewhere. I can look into it if it's important to address..it seemed kindof trivial though!

Reply:
It's really odd that Chromatic is not using the last generated storybook instance. I can also see what @marcysutton mentions.

- Chromatic is using this hash to run snapshots: `jxifgupglv`.
- The last site generated and uploaded to Chromatic is `jnmmnedmeq`.

Reply:
It's looking correct to me now on the Chromatic URL here in the PR: https://5e1bf4b385e3fb0020b7073c-jnmmnedmeq.chromatic.com/?path=/docs/overview--docs

Reply:
Agreed, I understand that (you can see the IDs/hashes included in the comment).

My worryness with this is that Chromatic is using the wrong snapshots for visual regression testing and this could be a possible caused for flaky-ness in the future.

Have you tried pushing more changes to see if the build for visual tests changes?

Reply:
Good question! I just pushed another update to override some Storybook styling, that depended on Base 8 font size. I'm not seeing the same layout shifts anymore, so we'll see if it helps the visual tests!

Reply:
YAY it works again, thanks for trying it out again :)